src/Cli/dotnet/NugetPackageDownloader/NuGetPackageDownloader.cs

```diff
@@ -157,18 +154,48 @@ private void VerifySigning(string nupkgPath)
                 return;
```

Comment:
I am not sure if this is the correct location to add the comment. It could be possible that I don't have a good understanding of the code in this repo. I think we should pass `verifySignatures to true` when trying to execute the dotnet tool install command.

https://github.com/JL03-Yue/sdk/blob/684d26597df5b1e06a799ce7481d97e6670fdd79/src/Cli/dotnet/commands/dotnet-tool/install/ToolInstallGlobalOrToolPathCommand.cs#L84

If we don't pass `verifySignatures = true` then I think the if block will return immediately there by skipping signature verification.

```cs
nugetPackageDownloader ??= new NuGetPackageDownloader(tempDir, verboseLogger: new NullLogger(), restoreActionConfig: restoreAction, verbosityOptions: _verbosity);
```